assets/javascripts/discourse/templates/admin-activity-pub-actor-show.hbs

```diff
@@ -0,0 +1,191 @@
+<div class={{this.containerClass}}>
+  <div class="admin-title activity-pub-actor-title">
+    <h2>{{titleLabel}}</h2>
+  </div>
+
+  <div class="activity-pub-actor-header">
+    {{#if this.actor.isNew}}
+      <div class="activity-pub-actor-setting activity-pub-new-actor-model">
+        <DropdownSelectBox
+          @nameProperty="label"
+          @value={{this.actor.model_type}}
+          @content={{this.modelTypes}}
+          @onChange={{action (mut this.actor.model_type)}}
+          class="activity-pub-model-type-chooser"
+          @options={{hash
+            none="admin.discourse_activity_pub.actor.model_type.none"
+          }}
+        />
```

Comment:
For a followup, since we're only going to have one of two options here (a category or a tag), I would prefer a radio button or something else here other than a dropdown. It's fine for now, though.

Reply:
It's possible there could also be Group (i.e. user group) and even custom Persons (i.e. users) here too. We may not have either, but other actor types are conceivable.